### ISSUE TITLE
Not works because the URL of `cadmium-playercore` has changed

### ISSUE THREAD

**Issue by fifv** [DavidBuchanan314/Turbo-Recadmiumator#18]
I'm using Firefox and suddenly it doesn't work, there are no console log at all.

After some source code reading and debugging I found that extension failed to inject code because the URL of `cadmium-playercore` becomes 
`https://assets.nflxext.com/player/html/ffe/cadmium-playercore-6.0047.317.911.js` 
which can't match current filter rules 
`"*://assets.nflxext.com/*/ffe/player/html/*",`

To fix that:

1. Add one rule to `manifest.json` > `permissions`
```diff
"permissions": [
    //...
+    "*://assets.nflxext.com/player/html/ffe/*",
    //...
]
```

2. Add one rult to `background.js` > `chrome.webRequest.onBeforeRequest.addListener` > `urls`
```diff
urls: [
    "*://assets.nflxext.com/*/ffe/player/html/*",
    //.....
+  "*://assets.nflxext.com/player/html/ffe/*",
]
```

After adding these, Netflix again plays in 1080p :)

I don't know if the changing of URL only happens to me as no one submit this issue...
Hope this can help you

**Comment by DavidBuchanan314** [comment]
Unless I'm misreading you, this was fixed all the way back in #14

**Comment by fifv** [comment]
Oh ... you are right! It's strange that I encountered this issue recently while the pr encountered it 2 months ago.
Nevermimd, thank you for response and have a nice day!
